### PR TITLE
Show model CoM

### PR DESCRIPTION
**Pull request by EnricoMingo**
The plugin permits to show & broadcast in YARP the position of the CoM of the robot wrt the (gazebo)world. The positon is broadcasted in yarp through a simple port. 
What is missing is: the "default" way to take the robot name from the config file (now the sdf name is used) and eventually a marker of the pose of the world in gazebo.

### CODE REVIEW

**Comment by francesco-romano** [comment]
Thanks @EnricoMingo .
@traversaro and me will take a look at this as soon as possible :)

File: CMakeLists.txt [hunk 1]
```diff
@@ -148,8 +156,10 @@ install(TARGETS gazebo_yarp_singleton
                 gazebo_yarp_imu
                 gazebo_yarp_jointsensors
                 gazebo_yarp_clock
-				gazebo_yarp_externalwrench
+                gazebo_yarp_externalwrench
+                gazebo_yarp_showmodelcom
                 gazebo_yarp_camera
+                gazebo_yarp_showmodelcom
```
Comment (traversaro) [inline CMakeLists.txt]: duplication?

Comment (EnricoMingo) [inline CMakeLists.txt]: ???
Il 21/mar/2015 10:47 "Silvio Traversaro" <notifications@github.com> ha
scritto:

**Comment by traversaro** [comment]
Merging, given that it is a different plugin we can fix all possible issues directly in master and we will not disturb any existing user.

File: CMakeLists.txt (hunk 1 above)
Comment (traversaro) [inline CMakeLists.txt]: fixed

**Pull request merged**